content/api/cli.md

```diff
@@ -195,38 +195,38 @@ These options allow webpack to display various stats and style them differently
 | Parameter               | Explanation                                                        | Type    |
 |-------------------------|--------------------------------------------------------------------|---------|
 | --color, --colors       | Enables/Disables colors on the console [default: (supports-color)] | boolean |
```

Comment:
This flag doesn't exist anymore if I remember right (handled automatically). Check this out from the source so we get it right.